docs(protocol-designer): PD 8.10 manual revisions (#21177)
OpenTrons/opentrons-api: docs/protocol-designer/docs/steps/transfer.md

Your protocol timeline includes steps the robot will peform in your protocol. To start, the timeline includes the starting and ending deck states. Click **Add Step** in the lower left to add transfer, move, mix, pause, or module-specific steps to your protocol. 

!!! Note

>     Lids on labware block transfer steps in your protocol. If every piece of labware on the deck includes a lid, only move, pause, and relevant module steps will be available. Use a move step to remove lids, then add a transfer step. 

Reviewer: For this, "relevant module steps" is it possible to list those or it doesn't make sense to do so here?

Author: I could list them, I would just have to list kind of every module. By "relevant module steps" I just mean that PD will show you the option to add a module-specific step for any module you have placed on the deck.

## The basics

Transfer steps move liquid from one well or group of wells to another. Adding a transfer step opens a four-part form. In the first, select basic settings for your liquid transfer: 

* The pipette to perform the transfer and the tip rack it will use. 
* Source and destination labware.
* The number of multi-channel pipette nozzles and labware wells to use during the transfer.
* Pipette path, or motion the pipette uses to perform the transfer. 

> * The volume of liquid to transfer.

Reviewer: The intro text opens itself up to quick obsolescence by declaring there's a 4 part form. If a new feature is added, does that make a 5 part form?

Also, this intro graf shows 5 bullets after the promise of a 4 part form. There might be some dissonance here.

As a possible solution, instead of a counting parts intro, consider rephrasing that to make this workflow a "process" which is a more generic approach. Here's an example:

> Transfer steps move liquid between wells or labware. When you add a transfer step, you begin a configuration process that defines some parameters...

or

> Transfer steps move liquid from one well or group of wells to another. Adding a transfer step opens a configuration form where you select the basic settings for the transfer:

This "form count" structure appears throughout. Rather than a line by line examination here, a broad observation that rather than describing the container (the form/parts), describe the content (the parameters/logic). Focus on what the user is defining or accomplishing. Using "form 1" "form 2" etc. forces the reader to keep a mental count of where they are in a procedure that might not even be visible as "1, 2, 3, 4" in the UI.

Author: Every time I think about changing this and then end up chickening out. 

In the PD manual, I try to use "menu" for something that pops up in the middle of the screen and "form" for something that appears on the right side to be consistent. I do think that if I keep this, we are in no danger of changing the # of forms in 2026. They'll be like this for a while (and have been around since the redesign).

Reviewer: I understand now, thanks for the explanation. Comment resolved!

### Wells and nozzles

To start, use the dropdown menus to select your source and destination labware. Next, click to select your pipette nozzles and wells. 

<figure class="screenshot" markdown>
  ![Select pipette nozzles to use in a transfer.](../images/wells_nozzles.png)
  <figcaption>Select the nozzles the Flex 96-channel pipette will use for the transfer.</figcaption>
</figure>

For Flex 96-channel and Flex or OT-2 8-channel pipettes, choose from single, column, or row nozzle configurations, depending on your attached pipettes. For more, see [partial tip pickup](transfer.md#partial-tip-pickup).

Next, select source and destination wells. Protocol Designer shows available wells based on your pipette nozzle selections. 

<figure class="screenshot" markdown>
  ![Select wells to aspirate liquid.](../images/aspirate_wells.png)
  <figcaption>Click to select wells a Flex 1-channel pipette will aspirate from.</figcaption>
</figure>

### Pipette path

> Pipette path and tip handling options, shown below, are only available if they are compatible with your well selections and other transfer settings. In the fourth form, you can also customize [tip management](transfer.md#tip-management) settings, like how often the pipette picks up a new tip.

Reviewer: Can we smooth out the intro some?

> Pipette path and tip handling options depend on your well selections and transfer settings. You can further customize how often the pipette picks up a new tip in the [Tip Management](https://www.google.com/search?q=transfer.md%23tip-management&authuser=1) section.

Reviewer 2: agreed

Author: Good suggestion; thanks! I updated the first sentence, but left the "fourth form" text in here so that it follows/lets users see the flow of decisions in PD.

Your protocol timeline includes steps the robot will peform in your protocol. To start, the timeline includes the starting and ending deck states. Click **Add Step** in the lower left to add transfer, move, mix, pause, or module-specific steps to your protocol. 

!!! Note
    Lids on labware block transfer steps in your protocol. If every piece of labware on the deck includes a lid, only move, pause, and relevant module steps will be available. Use a move step to remove lids, then add a transfer step. 

## The basics

Transfer steps move liquid from one well or group of wells to another. Adding a transfer step opens a four-part form. In the first, select basic settings for your liquid transfer: 

* The pipette to perform the transfer and the tip rack it will use. 
* Source and destination labware.

> * The number of multi-channel pipette nozzles and labware wells to use during the transfer.

Reviewer 2: You can just say "The number of pipette nozzles and labware wells to use during the transfer." because  this modal pops up for single channel pipettes as well.

* Pipette path, or motion the pipette uses to perform the transfer. 
* The volume of liquid to transfer.

### Wells and nozzles

To start, use the dropdown menus to select your source and destination labware. Next, click to select your pipette nozzles and wells. 

<figure class="screenshot" markdown>
  ![Select pipette nozzles to use in a transfer.](../images/wells_nozzles.png)
  <figcaption>Select the nozzles the Flex 96-channel pipette will use for the transfer.</figcaption>
</figure>

> For Flex 96-channel and Flex or OT-2 8-channel pipettes, choose from single, column, or row nozzle configurations, depending on your attached pipettes. For more, see [partial tip pickup](transfer.md#partial-tip-pickup).

Reviewer 3: ```suggestion
Choose from single, column, or row nozzle configurations, depending on your attached pipettes. For more, see [partial tip pickup](transfer.md#partial-tip-pickup).
```
Simpler, and doesn't lead to any reading where you might (even briefly) think that row layout applied to 8-channel pipettes.

| Delay | <ul><li>Aspirate</li><li>Dispense</li></ul> | <ul><li>Hold the pipette tip at the submerge, aspirate or dispense, or retract position for a defined amount of time after aspirating or dispensing</li><li>Customize duration and position from bottom of well</li></ul> |
| Condition | <ul><li>Aspirate</li></ul> | <ul><li>Aspirate a small conditioning volume after aspirating the total volume to be transferred</li><li>Pipetted back into the source for a more accurate first dispense</li><li>Only available for distribute, or multi-dispenses</li></ul> |
| Push out | <ul><li>Dispense</li><ul> | <ul><li>Dispense past the pipette's plunger bottom to ensure all liquid leaves the tip</li><li>Customize volume</li></ul> |

> | Blowout | <ul><li>Dispense</li></ul> | <ul><li>Blow any remaining liquid out of the tip</li><li>Customize location (source well, destination well, trash bin, or trash chute)</li><li>Customize flow rate and tip position from bottom during blowout</li><li>If you'll blow out liquid into a source or destination well, customize blowout start point</li></ul> |

Reviewer 3: Second person is a little odd in this list.

```suggestion
| Blowout | <ul><li>Dispense</li></ul> | <ul><li>Blow any remaining liquid out of the tip</li><li>Customize location (source well, destination well, trash bin, or trash chute)</li><li>Customize flow rate and tip position from bottom during blowout</li><li>Customize blowout start point, if blowing out liquid into a source or destination well.</li></ul> |
```

## Partial tip pickup

You can use partial tip pickup to use less tips than a Flex or OT-2 multi-channel pipette can pick up at once. When you add a transfer or mix step, start by clicking **Pipette nozzles and wells** in the first form. Select an available partial tip nozzle option: 

| **Pipette** | **Available Nozzle Configurations** |
| ----------- | ----------------------------------- |  

> | Flex 96-channel | <ul><li>All nozzles (recommended)</li><li>Single nozzle (A1, A12, H1, or H12)</li><li>Single column (column 1 or 12)</li><li>Single row (row A or H)</li></ul> |

Reviewer 3: I think it says "recommended" in the UI but I've always found that a bit odd (it might be the opposite of what you need!). We could omit it here.

Author: Yeah, I agree. The UI is persistent with the "always recommended," but I'm happy to omit it from the table. I have some text below that states that PD will always recommend you use all available nozzles.

| Flex 8-channel | <ul><li>All nozzles (recommended)</li><li>Single nozzle (A1 or H1)</li><li>Partial nozzles (2–7 consecutive nozzles)</li></ul> |
| OT-2 8-channel | <ul><li>All nozzles (recommended)</li><li>Single nozzle (A1 or H1)</li><li>Partial nozzles (2–7 consecutive nozzles)</li></ul> |

> Here, choose a single column of nozzles ( 1 or 12) for the Flex 96-channel pipette to use in partial tip pickup.  

Reviewer 3: ```suggestion
The example below uses a single column of nozzles (1 or 12) for the Flex 96-channel pipette to use in partial tip pickup.  
```

In the fourth form, you can choose between automatic and manual tip tracking for your multi-channel pipette. Protocol Designer includes warnings, both in the tip selection form and as a step error, if your partial tip selections could cause pipette collisions. 

### Collision errors

Partial tip pickup can increase the risk of collisions, so Protocol Designer will always recommend selecting the total number of nozzles for your attached pipette.

When you choose a partial tip pickup nozzle configuration, Protocol Designer includes [warnings and errors](warnings-errors.md) to reduce collision risk. You also won't be able to select incompatible wells and tips, where pickup could cause a collision with adjacent labware or modules. 

Partial tip pickup requires the pipette to hover above adjacent deck slots. Here, Protocol Designer won't allow you to select tips for manual tip tracking, because the tip racks in adjacent deck slots could cause a pipette collision.

<figure class="screenshot" markdown>
  ![Pipette collision risk](../images/partial-tip-collision.png)
  <figcaption>Labware in adjacent slots can cause pipette collisions.</figcaption>
</figure> 

> Despite the tip rack in the example above being full of pipette tips, the "not enough tips" warning tells you that there aren't enough pipette tips that the pipette can pick up safely. This also results in a pipette collision error. 

Reviewer: This might be a long way around into the heart of the paragraph:

> Despite the tip rack in the example above being full of pipette tips

and could be removed by going straight at it:

> Even if a tip rack is full, the "not enough tips" warning appears if the pipette cannot safely access them. This <state> <condition> <situation> triggers a collision error to prevent the pipette from striking adjacent labware.

Author: I like this; made some changes here!

> Protocols containing errors cannot be run on the robot and will cause a run to fail. You'll need to resolve these errors before exporting your protocol. 

Reviewer: Some redundancies here:
- "Cannot be run"
- "cause a run to fail"

> To ensure your protocol runs successfully, fix all errors before exporting. Flex will not run protocols with unresolved errors.

or something something

Author: Updated to `Fix all errors before exporting to ensure your protocol runs successfully. You won't be able to run protocols with unresolved errors.`

- avoid using "to" to start multiple sections under Pipette collisions
- avoid being specific for just Flex

In the fourth form, you can choose between automatic and manual tip tracking for your multi-channel pipette. Protocol Designer includes warnings, both in the tip selection form and as a step error, if your partial tip selections could cause pipette collisions. 

### Collision errors

> Partial tip pickup requires the pipette to hover above adjacent deck slots, and can increase the risk of collisions.Protocol Designer will always recommend selecting the total number of nozzles for your attached pipette. 

Reviewer 3: ```suggestion
Partial tip pickup requires the pipette to hover above adjacent deck slots, and can increase the risk of collisions. Protocol Designer will always recommend selecting the total number of nozzles for your attached pipette. 
```

When you choose a partial tip pickup nozzle configuration, Protocol Designer includes [warnings and errors](../warnings-errors.md) to reduce collision risk. These errors can require you to change your well, tip, or deck selections in order to avoid risky situations. 

Here, Protocol Designer won't allow you to select tips for manual tip tracking, because the tip racks in adjacent deck slots could cause a pipette collision.

<figure class="screenshot" markdown>
  ![Pipette collision risk](../images/partial-tip-collision.png)
  <figcaption>Labware in adjacent slots can cause pipette collisions.</figcaption>

> </figure> 

Reviewer 3: I find the "Here, " construction a little confusing…I don't naturally look down to the next figure.

I think this one can be solved by moving the details into the caption.

```suggestion
<figure class="screenshot" markdown>
![Pipette collision risk](../images/partial-tip-collision.png)
<figcaption>Protocol Designer showing a collision error, because the tip rack in C2 blocks partial tip pickup from the rack in C3.</figcaption>
</figure> 
```

Even though the tip rack above is full of pipette tips, the "not enough tips" warning tells you that there aren't enough tips that the pipette can safely access. This results in a collision error to prevent the pipette from striking adjacent labware in your final protocol. 

> To fix the collision error, move the tip racks from adjacent deck slots or change your tip pickup selections.

Reviewer 3: ```suggestion
Even though the tip rack above is full of pipette tips, the "not enough tips" warning tells you that there aren't enough tips that the pipette can safely access.

To fix the errors, move the tip racks so they are no longer adjacent, or change your tip pickup selections.
```
